src/test/java/org/hisp/dhis/rules/ConstantsValueTests.java

```diff
@@ -94,12 +92,13 @@ public void assignConstantValueFromAssignActionInEnrollment() throws Exception
     @Test
     public void assignConstantValueFromAssignActionInEvent() throws Exception
     {
-        RuleAction assignAction = RuleActionAssign.create(null, "C{test_constant_value}", "test_data_element");
+        RuleAction assignAction = RuleActionAssign.create(null, "C{A1234567890}", "${test_data_element}");
```

Comment:
$ syntax is not valid in rule engine. Would not this rule fail?
Correct syntax would be #{test_data_element}

Reply:
I have tested this function. It passes with all these values. I think it should fail.
```
RuleAction assignAction = RuleActionAssign.create(null, "C{test_constant_value}", "A{test_data_element}");
RuleAction assignAction = RuleActionAssign.create(null, "C{test_constant_value}", "#{test_data_element}");
RuleAction assignAction = RuleActionAssign.create(null, "A{test_constant_value}", "A{test_data_element}");
```

Reply:
This is just following the pre-existent behaviour.
The third parameter of the RuleActionAssign create method is the field, so it is not going to be evaluate by the grammar, but it is "unwrapped" by a method in RuleEngineExecution.

How should be the format of such a field?